platform-services.md

```diff
@@ -51,17 +51,17 @@ non-graphical, data-only information. Examples include:
 *   Network information
 *   Sensors
 
-You can also use the [`HostMessages`](https://docs.flutter.io/flutter/services/HostMessages-class.html)
+You can also use the [`PlatformMessages`](https://docs.flutter.io/flutter/services/PlatformMessages-class.html)
 class to invoke APIs from third-party SDKs that run on iOS and Android devices.
 
 To communicate between the
-[`HostMessages`](https://docs.flutter.io/flutter/services/HostMessages-class.html) class and
+[`PlatformMessages`](https://docs.flutter.io/flutter/services/PlatformMessages-class.html) class and
 platform-specific APIs in the host, you must add the following Flutter components into
 the host's view hierarchy.
 
-+ **On iOS:** Add a [`FlutterViewController`](https://github.com/flutter/engine/blob/master/sky/shell/platform/ios/framework/Headers/FlutterViewController.h)
++ **On iOS:** Add a [`FlutterViewController`](https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h)
```

Comment:
We may move files around in the repository. If we ever moved FlutterViewController, it would result in a broken link on the website. A handy trick on GitHub it to press `y`. This will replace the current link (the one containing `blob/master`) with a link that explicitly specifies the SHA (`blob/<SHA>`). We'd still need to update the website but at least users won't see broken links in the meantime. And updating docs would be as simple as doing as looking for all instances of that SHA.